src/screen-reader-only.css.ts

```diff
@@ -13,9 +13,8 @@ export const screenReaderOnly = style([
         width: 1,
         height: 1,
         clip: 'rect(0, 0, 0, 0)',
+        clipPath: 'inset(50%)',
+        whiteSpace: 'nowrap',
```

Comment:
I've found other libs include this too.
Some references:
https://kittygiraudel.com/snippets/sr-only-class/
https://tailwindcss.com/docs/screen-readers